.gitignore

```diff
@@ -127,3 +128,6 @@ compilation-stats.json
 
 # auto generated frontend docs
 /docs/sources/packages_api
+
+# auto generated Go files
+*_gen.go
```

Comment:
🤔 looks related to my question about CI - are we not including generated code in git?

Reply:
Not currently because OSS and Enterprise writes generated code to the same file. Maybe something we can look into in a separately after this been merged?